Bai_Tap/Buoi3_Bai4.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title></title>
+    <style>
+      .content {
+        padding-left: 20px;
+        padding-top: 20px;
+        padding-right: 20px;
+        padding-bottom: 20px;
+      }
+
+      .kth {
+        width: 20px;
+        height: 16px;
+      }
+    </style>
+  </head>
+
+  <body>
+    <p>Information</p>
+    <div class="content">
+      <form>
+        <label for="search">Search:</label>
+        <input
+          type="text"
+          id="search"
+          name="search"
+          placeholder="Enter keyword..."
+        /><br />
+        <label for="url">Url path:</label>
+        <input type="url" id="url" name="url" /><br />
+        <label for="user">User:</label>
+        <input
+          type="text"
+          id="user"
+          name="user"
+          placeholder="Enter Username..."
+        /><br />
+        <label for="date">Date:</label>
+        <input type="date" id="date" name="date" /><br />
+        <label for="email">Email:</label>
+        <input
+          type="email"
+          id="email"
+          name="email"
+          placeholder="Enter Email..."
+        /><br />
+        <label for="pass">Pass:</label>
+        <input
+          type="password"
+          id="pass"
+          name="pass"
+          placeholder="Enter password..."
+        /><br />
+        <label for="cm">Comment:</label><br />
+        <textarea
+          type="message"
+          id="cm"
+          name="cm"
+          cols="30"
+          rows="10"
+          placeholder="comment here..."
+        ></textarea>
+      </form>
+    </div>
+
+    <div>
+      <form>
+        <fieldset>
+          <legend>Controls</legend>
+        <p for="phai">
+          Genner:
+          <input type="radio" id="nam" name="phai" />
+          <label for="nam"> Nam </label>
+          <input type="radio" id="nu" name="phai" />
+          <label for="nu">Nữ</label>
+          <input type="radio" id="khac" name="phai" checked="true" />
+          <label for="khac">Khác</label>
+        </p>
+        <p>
+          So thich:
+          <input type="checkbox" checked="true"/>
+          Khong so thich
```

Comment:
ref: https://www.w3schools.com/tags/att_input_type_checkbox.asp
```suggestion
         <input type="checkbox" id="khongsothich" name="khongsothich" value="Khong so thich" checked >
          <label for="khongsothich"> Khong so thich</label><br> 
```